addons/bestja_page_fixtures_ucw/templates/pages/home_page.xml

```diff
@@ -39,8 +39,8 @@
                                                 </div>
                                                 <div class="carousel_title">
                                                     <h1>
-                                                        Use your professional skills <br />
-                                                        to make a difference2
+                                                        <a class="color_white" href="#choose_what_to_do">Use your professional skills <br />
```

Comment:
The same issue. Class name `color_white` is unacceptable. You are completely mixing your content layer (HTML) and presentation layer (CSS). You should named the class based on its *function* ("carousel_link"? you can also avoid naming it completely and use ". carousel_title a" in your css) in HTML and than define its appearance in CSS. 

When you use classes like "color_white", instead of defining the appearance in CSS you define it in HTML. You also block yourself from having the ability to easily change the appearance of the element in CSS in the future. It's like you went back to the 90s and invented `<font color="while">` all over again.

See also: http://en.wikipedia.org/wiki/Separation_of_concerns